appveyor.yml

```diff
@@ -4,9 +4,9 @@ environment:
   node_pre_gyp_secretAccessKey:
     secure: BFW25HQ0Jdq8IpysC59Y2mh1J9IsXwWB+FTElHVSYVl5xDwwZPHrXwctZQLzKrw9
   matrix:
-    - nodejs_version: 6
     - nodejs_version: 8
     - nodejs_version: 10
+    # - nodejs_version: 12
```

Comment:
```suggestion
     - nodejs_version: 12
```

Reply:
I tried that earlier, but appveyor does not have Node 12 "by default" yet (check Node 12 build errors at https://ci.appveyor.com/project/mcollina/native-hdr-histogram/builds/24526661).

We would have to change build script like that:
https://github.com/appveyor/ci/issues/2921#issuecomment-486727727
but i wasn't sure it would be ok.

Reply:
Go for it

Reply:
Done.

Node v8, v10, v12 on Linux, OSX and Windows are all green now :D.